app/renderers/building.ts

```diff
@@ -28,7 +28,7 @@ export default class BuildingRenderer extends PIXI.Graphics {
       }
       case Building.Academy1:
       case Building.Academy2: {
-        this.drawCircle(0, 0, 0.5*scale);
+        this.drawRect(-0.3*scale, -0.3 * scale, 0.6 * scale, 0.6 * scale);
```

Comment:
Why?

Reply:
I think I've messed up a merge due to conflicts, as I told you a the beginning of this PR. Sorry!